cmd/orderimports/orderimports.go

```diff
@@ -0,0 +1,226 @@
+			var localImports []string
+			var k8sImports []string
+			var externalImports []string
+
+			for _, imp := range file.Imports {
+				importPath := strings.Replace(imp.Path.Value, "\"", "", -1)
+				parts := strings.Split(importPath, "/")
+				originalImports = append(originalImports, importPath)
+
+				if !strings.Contains(parts[0], ".") {
+					// standard library imports are first
+					stdlibImports = append(stdlibImports, importPath)
+				} else if strings.HasPrefix(importPath, "k8s.io/kubernetes") {
+					// local imports are second
+					localImports = append(localImports, importPath)
+				} else if strings.Contains(parts[0], "k8s.io") {
+					// other *.k8s.io imports are third
+					k8sImports = append(k8sImports, importPath)
+				} else {
+					// external repositories are fourth
+					externalImports = append(externalImports, importPath)
+				}
+
+			}
+			if len(originalImports) == 0 {
+				continue
+			}
+			sort.Strings(stdlibImports)
+			sort.Strings(localImports)
+			sort.Strings(k8sImports)
+			sort.Strings(externalImports)
+
+			orderImports := []string{}
+			orderImports = append(orderImports, stdlibImports...)
+			orderImports = append(orderImports, localImports...)
+			orderImports = append(orderImports, k8sImports...)
+			orderImports = append(orderImports, externalImports...)
+			if diff := cmp.Diff(originalImports, orderImports); diff != "" {
+				a.failed = true
+				fmt.Printf("%s (-got +want):\n%s", pathToFile, diff)
+			}
+		}
+	}
+}
+
+func (a *analyzer) filterFiles(fs map[string]*ast.File) []*ast.File {
+	var files []*ast.File
+	for fileName, f := range fs {
+		if a.ignore.MatchString(fileName) {
+			continue
+		}
+		files = append(files, f)
+	}
+	return files
+}
+
+type collector struct {
+	dirs  []string
+	regex *regexp.Regexp
+}
+
+// handlePath walks the filesystem recursively, collecting directories,
+// ignoring some unneeded directories (hidden/vendored) that are handled
+// specially later.
+func (c *collector) handlePath(path string, info os.FileInfo, err error) error {
+	if err != nil {
+		return err
+	}
+	if info.IsDir() {
+		// Ignore hidden directories (.git, .cache, etc)
+		if len(path) > 1 && path[0] == '.' ||
+			// Staging code is symlinked from vendor/k8s.io, and uses import
+			// paths as if it were inside of vendor/. It fails typechecking
+			// inside of staging/, but works when typechecked as part of vendor/.
+			path == "staging" ||
+			// OS-specific vendor code tends to be imported by OS-specific
+			// packages. We recursively typecheck imported vendored packages for
+			// each OS, but don't typecheck everything for every OS.
+			path == "vendor" ||
+			path == "_output" ||
+			// This is a weird one. /testdata/ is *mostly* ignored by Go,
+			// and this translates to kubernetes/vendor not working.
+			// edit/record.go doesn't compile without gopkg.in/yaml.v2
+			// in $GOSRC/$GOROOT (both typecheck and the shell script).
+			path == "pkg/kubectl/cmd/testdata/edit" {
```

Comment:
Did you manage to investigate if we can remove this one?

Reply:
Yes, it can be removed. `pkg/kubectl/cmd/testdata/edit` was only used in the early days 

https://github.com/kubernetes-csi/driver-registrar/tree/master/vendor/k8s.io/kubernetes/pkg/kubectl/cmd/testdata/edit